repokid/commands/role.py

```diff
@@ -235,22 +236,25 @@ def _display_role(
     for permission in permissions:
         service = permission.split(":")[0]
         action = permission.split(":")[1]
-        repoable = permission in role.repoable_services
+        repoable = (
```

Comment:
nit: Can you add a comment here or somewhere else with some example values for role.repoable_services?

I'm imagining a list of permissions like ["sqs:ListQueue", "sns:ListTopic"] , but I'm unsure. 

When would the extra condition be used? `or permission.split(":")[0] in role.repoable_services`

Reply:
Yeah, I'll add something. This logic is super convoluted and needs refactoring. I carried it over from the previous implementation to ensure consistency until testing is more robust.

Reply:
Oops, I thought this comment was somewhere else.

`role.repoable_services` is both repoable actions AND services. The colon split is used to check if the service name is in the list. It's gross, but I haven't gotten around to refactoring it yet.

Reply:
Alright, I broke up the logic and added a comment to explain what's going on here